README.md

```diff
@@ -102,4 +128,81 @@ go test -v ./test/integration --args -config=<path to config.json>
+
+#### Feature Requests
+
+1. Before submitting a feature request, again, check the [existing issues](https://github.com/icon-project/IBC-Integration/issues) to ensure that the feature hasn't already been requested. If you find a similar request, you can add your insights in the comments.
+
+2. To submit a new feature request, [open a new issue](https://github.com/icon-project/IBC-Integration/issues/new/choose) and select the "Feature Request" template. Provide a clear and comprehensive description of the feature you're proposing, including:
+   - The problem the feature aims to solve.
+   - How the feature would work.
+   - Any potential benefits or use cases.
+
+3. Assign appropriate labels to the issue, such as "enhancement" or "feature request," and any other relevant tags.
+
+### Pull Requests
+
+If you're interested in contributing code to the project, follow these steps to submit a pull request (PR):
+
+1. **Fork the Repository:** Click the "Fork" button at the top of the [repository page](https://github.com/icon-project/IBC-Integration) to create your own fork of the project. This will allow you to work on your changes in your own copy of the repository.
+
+2. **Create a Branch:** Create a new branch in your forked repository that will contain your changes. Naming the branch according to the changes you're making is a good practice (e.g., `feature/new-feature` or `bugfix/fix-issue-123`).
+
+3. **Make Changes:** Make your changes in the new branch. Follow the project's coding standards, and write clear and concise commit messages.
+
+4. **Test Your Changes:** If applicable, ensure that your changes work as intended and do not introduce new bugs. If tests exist in the project, make sure to run them.
+
+5. **Open a Pull Request:** When you're ready to submit your changes, [open a new pull request](https://github.com/icon-project/IBC-Integration/compare) from your branch to the `main` branch of the main repository. Provide a detailed description of your changes, including the problem you're solving, your solution, and any relevant context.
+
+6. **Review and Iteration:** Your pull request will be reviewed by the project maintainers. Be prepared to address any feedback or changes requested by the reviewers. Collaboration and constructive discussion are key.
+
+7. **Merge:** Once your pull request passes the review process and meets the project's standards, it will be merged into the `main` branch.
+
+### Code of Conduct
+
+Contributors are expected to adhere to the project's [Code of Conduct](CODE_OF_CONDUCT.md) at all times. This ensures a respectful and inclusive environment for all contributors and participants.
+
+We appreciate your dedication to contributing to the IBC Integration for ICON Project. Your efforts help improve the project's quality and expand its capabilities. Thank you for being a part of our community!
+
+
+## License <a name = "license"> </a>
+
+This project is licensed under the [MIT License](LICENSE).
```

Comment:
MIT or Apache 2.0? I think it is an Apache 2.0 license.

Reply:
It is Apache i have updated the link